tests/end-to-end/regression/2155-no-error_log.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
```

Comment:
`expectErrorLog()` is a new feature. Therefore, we should not put its end-to-end tests into the `regression` directory. The `generic` would be acceptable for now.